.fixtures.yml

```diff
@@ -1,8 +1,8 @@
 fixtures:
-  forge_modules:
-    apt: 'puppetlabs-apt'
-    augeas: 'camptocamp-augeas'
-    augeasproviders_core: 'herculesteam-augeasproviders_core'
-    stdlib: 'puppetlabs-stdlib'
+  repositories:
```

Comment:
Got it, thanks.

On Wed, Aug 31, 2016 at 7:03 PM, Tim Meusel <notifications@github.com>
wrote:

> In .fixtures.yml
> <https://github.com/voxpupuli/puppet-puppetserver/pull/30#discussion_r77089392>
> :
>
> > @@ -1,8 +1,8 @@
> >  fixtures:
> > -  forge_modules:
> > -    apt: 'puppetlabs-apt'
> > -    augeas: 'camptocamp-augeas'
> > -    augeasproviders_core: 'herculesteam-augeasproviders_core'
> > -    stdlib: 'puppetlabs-stdlib'
> > +  repositories:
>
> because we do that in every other module too. We always test against
> master, not the last release.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/voxpupuli/puppet-puppetserver/pull/30/files/21dd3f64b64b8efbf286da06f94ce1d5645dabb5#r77089392>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPdK5F13Uj9ejEbLbKAkESN0oflQAobks5qlggygaJpZM4JcCsX>
> .
>